src/panels/config/automation/trigger/types/ha-automation-trigger-tag.ts

```diff
@@ -0,0 +1,37 @@
+import "@polymer/paper-input/paper-input";
+import { customElement, html, LitElement, property } from "lit-element";
+import { TagTrigger } from "../../../../../data/automation";
+import { HomeAssistant } from "../../../../../types";
+import {
+  handleChangeEvent,
+  TriggerElement,
+} from "../ha-automation-trigger-row";
+
+@customElement("ha-automation-trigger-tag")
+export class HaTagTrigger extends LitElement implements TriggerElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property() public trigger!: TagTrigger;
+
+  public static get defaultConfig() {
+    return { tag_id: "" };
+  }
+
+  protected render() {
+    const { tag_id } = this.trigger;
+    return html`
+      <paper-input
```

Comment:
We should make this a dropdown with all possible tags.

Reply:
Yes, wanted to keep it simple for now